internal/service/studies/main.go

```diff
@@ -0,0 +1,36 @@
+package studies
+
+import (
+	"github.com/google/uuid"
+	"github.com/ucl-arc-tre/portal/internal/graceful"
+	"github.com/ucl-arc-tre/portal/internal/types"
+	"gorm.io/gorm"
+)
+
+type Service struct {
+	db *gorm.DB
+}
+
+func New() *Service {
+	return &Service{
+		db: graceful.NewDB(),
+	}
+}
+
+// GetStudyAssets retrieves all assets for a study,
+// ensures that the user owns the study (this might be refactored later to allow shared access)
+func (s *Service) GetStudyAssets(studyID uuid.UUID, userID uuid.UUID) ([]types.Asset, error) {
+	// verify the user owns the study
+	var study types.Study
+	if err := s.db.Where("id = ? AND owner_user_id = ?", studyID, userID).First(&study).Error; err != nil {
+		return nil, err
+	}
```

Comment:
So I guess other users can access studies that they have been assigned to? I was waiting to see what the implementation was going to be in #152 before proceeding.